pkg/operator/bootstrapteardown/waitforceo.go

```diff
@@ -0,0 +1,71 @@
+package bootstrapteardown
+
+import (
+	"context"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	operatorversionedclient "github.com/openshift/client-go/operator/clientset/versioned"
+	operatorv1helpers "github.com/openshift/library-go/pkg/operator/v1helpers"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/cache"
+	clientwatch "k8s.io/client-go/tools/watch"
+	"k8s.io/klog"
+)
+
+func WaitForEtcdBootstrap(ctx context.Context, config *rest.Config) error {
+	operatorConfigClient, err := operatorversionedclient.NewForConfig(config)
+	if err != nil {
+		return err
```

Comment:
since we disregard this error should we print it as level 4 for debug?

Reply:
ack, will add logging